src/components/Details.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 
 export default class Details extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
+    }
+
+    componentDidMount() {
+        this.timerID = setInterval(
```

Comment:
2 things here:
1. You're using `setInterval`, which means the function will run, infinitely, every 3 seconds. We want to run it just once, using `setTimeout` instead
2. You are saving a variable `timerID`, but you aren't using it at all. Hence, I believe we don't need it.